### PR TITLE
fix IconButton in v5/v6

### PR DESCRIPTION
**Pull request by mshwery**
## Overview
This fixes the IconButton appearance/intent for v6 and v5 theming

## Screenshots (if applicable)
Screen Shot 2020-09-11 at 6 28 10 PM
Screen Shot 2020-09-11 at 6 28 13 PM

## Testing

- [ ] Updated Typescript types and/or component PropTypes
- [ ] Added / modified component docs
- [ ] Added / modified Storybook stories

### CODE REVIEW

File: src/select-menu/src/SelectMenuContent.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ const DefaultTitleView = ({ close, headerHeight, title }) => (
     <IconButton
       icon={CrossIcon}
       appearance="minimal"
-      height={24}
+      size="small"
```
Comment (akleiner2) [inline src/select-menu/src/SelectMenuContent.js:27]: Should we decouple this from the actual theme?

Comment (mshwery) [reply src/select-menu/src/SelectMenuContent.js:27]: Hm... curious what you mean?

Comment (mshwery) [reply src/select-menu/src/SelectMenuContent.js:27]: We could use `height` internally, since we dont know what's in the theme.

Comment (akleiner2) [reply src/select-menu/src/SelectMenuContent.js:27]: Yeah, think we should use `height` directly instead. This doesn't need to respect the theme imo.

**Pull request merged**